<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@838.va_3a_087b_4055b">
  <node class="org.jenkinsci.plugins.workflow.graph.FlowEndNode" plugin="workflow-api@1192.v2d0deb_19d212">
    <parentIds>
      <string>36</string>
    </parentIds>
    <id>37</id>
    <startId>2</startId>
    <result>
      <name>ABORTED</name>
      <ordinal>4</ordinal>
      <color>ABORTED</color>
      <completeBuild>false</completeBuild>
    </result>
  </node>
  <actions>
    <wf.a.ErrorAction plugin="workflow-api@1192.v2d0deb_19d212">
      <error class="org.jenkinsci.plugins.workflow.steps.FlowInterruptedException" plugin="workflow-step-api@639.v6eca_cd8c04a_a_">
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.steps.BodyExecution.cancel(BodyExecution.java:59)</trace>
          <trace>org.jenkinsci.plugins.workflow.steps.TimeoutStepExecution.cancel(TimeoutStepExecution.java:197)</trace>
          <trace>jenkins.security.ImpersonatingScheduledExecutorService$1.run(ImpersonatingScheduledExecutorService.java:69)</trace>
          <trace>java.base/java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)</trace>
          <trace>java.base/java.util.concurrent.FutureTask.run(Unknown Source)</trace>
          <trace>java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(Unknown Source)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)</trace>
          <trace>java.base/java.lang.Thread.run(Unknown Source)</trace>
        </stackTrace>
        <suppressedExceptions class="empty-list"/>
        <result>
          <name>ABORTED</name>
          <ordinal>4</ordinal>
          <color>ABORTED</color>
          <completeBuild>false</completeBuild>
        </result>
        <causes class="java.util.Arrays$ArrayList">
          <a class="jenkins.model.CauseOfInterruption-array">
            <org.jenkinsci.plugins.workflow.steps.TimeoutStepExecution_-ExceededTimeout plugin="workflow-basic-steps@994.vd57e3ca_46d24">
              <nodeId>26</nodeId>
            </org.jenkinsci.plugins.workflow.steps.TimeoutStepExecution_-ExceededTimeout>
          </a>
        </causes>
        <actualInterruption>false</actualInterruption>
      </error>
    </wf.a.ErrorAction>
    <wf.a.TimingAction plugin="workflow-api@1192.v2d0deb_19d212">
      <startTime>1663755582510</startTime>
    </wf.a.TimingAction>
  </actions>
</Tag>